datasets/de/abgeordnetenwatch/de_abgeordnetenwatch.yml

```diff
@@ -0,0 +1,21 @@
+name: de_abgeordnetenwatch
+title: "abgeordnetenwatch"
+url: https://www.abgeordnetenwatch.de
+description: >
+  On abgeordnetenwatch.de (which translates as “member of parliament watch”)
+  users can find all their representatives and ask them public questions, read
+  our investigative researchs and support our demands towards politics for more
+  transparency.
+entry_point: crawler
+prefix: de-aw
+publisher:
+    name: Parlamentwatch e.V.
+    organization: Parlamentwatch e.V.
+    authority: Deutscher Bundestag
```

Comment:
I don't see good documentation for this field other than the tutorial so I suspect this is for when the publisher is a unit within an official authority, which is not the case here. Or have you seen documentation to the contrary? I'd suggest we delete it for now while I sort out the metadata documentation.

```suggestion
```

Reply:
> Or have you seen documentation to the contrary?

No, I just assumed that the publisher would be the responsible to the data source url. Your suggestion is to delete the full section `publisher` or just `authority`. I got confused by the comment and github 'Suggested change'.

Reply:
Just authority. The rest of your publisher is good